scalableminds/webknossos
# show user activate button to orga-team managers

I have thought a little more in-depth about what should be permitted here to whom. Team Managers *of the organization team* are allowed to activate new users. Or, more generally speaking, Team Managers of a team that the user is in, should be allowed to activate them. And for new users that happens to be only the organization team.
Only admins should be allowed to deactivate users.

The backend already behaves this way, the frontend showed inconsistent information in the new-users banner vs the (editable) user list. This PR changes the list’s behavior to make everything consistent again.

### URL of deployed dev instance (used for testing):
- https://showactivatebuttontoteammanager.webknossos.xyz

### Steps to test:
- register two more accounts, let’s call them A and B
- activate A and promote A to teammanager of the organization team
- log in as A, navigate to user list. A should see B in the new-users-banner and in the list (when not filtered for active), and the list should also feature the »Activate User« button
- A should be allowed to activate B

- second scenario: as admin, create separate team. promote A to team manager of that team (but revoke their teammanager licence of the orga team)
- as admin, remove B from all teams except the orga team and deactivate B (this is the initial state after registering)
- login as A, A should now neither see B in the new-users banner nor in the list of users, even when not filtered for active

### Issues:
- fixes #3091 

------
